### ISSUE TITLE
Read-IcingaMSIMetadata not available during kickstart

### ISSUE THREAD

**Issue by t3easy** [Icinga/icinga-powershell-framework#522]
I tried to install two servers with the Kickstart script `https://packages.icinga.com/IcingaForWindows/IcingaForWindows.ps1``

Both can`t install the agent

```
[Notice]: Remote repository "Icinga Stable" was successfully added
[Notice]: Downloading "agent" from "https://packages.icinga.com/IcingaForWindows/stable/agent/Icinga2-v2.13.3-x86_64.msi"
powershell.exe : Die Benennung "Read-IcingaMSIMetadata" wurde nicht als Name eines Cmdlet, einer Funktion, einer
Skriptdatei oder eines ausführbaren Programms erkannt. Überprüfen Sie die Schreibweise des Namens, oder ob der Pfad
korrekt ist (sofern enthalten), und wiederholen Sie den Vorgang.
In C:\Program Files\WindowsPowerShell\Modules\icinga-powershell-framework\cache\framework_cache.psm1:16358 Zeichen:20
+         $MSIData = & powershell.exe -Command {
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Read-IcingaMSIMetadata:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

And then fail installing the certificate and so on.

Both already got a personal API token.

After closing the Admin Powershell, starting a new one and run the Kickstart a 2nd time, the installation worked fine.

**Comment by t3easy** [comment]
How can I assist here?

**Comment by LordHepipud** [comment]
Thank you for the report. This will be fixed with v1.10.0
